HtmlGenerators/RadiosHtmlGenerator.cs

```diff
@@ -37,6 +38,12 @@ internal static class RadiosHtmlGenerator
                 {
                     bool isEnumValueCurrentlySelected = selectedValue.HasValue && enumValue.Equals(selectedValue.Value);
                     string radioLabelText = GovUkRadioCheckboxLabelTextAttribute.GetLabelText(enumValue);
+                    HintViewModel itemHint = null;
+
+                    if (radioHints != null)

```

Comment:
I think this could be `radioHints?.TryGetValue(enumValue, out itemHint);`

Reply:
Done, thanks